pkg/engine/funcs_test.go

```diff
@@ -6,17 +6,21 @@ import (
 	"testing"
 )
 
+var empty_input = "empty input"
+var valid_input = "valid input"
+var invalid_input = "invalid input"
```

Comment:
Could you please move these variables into `TestToYAML` function and use camel-case? You may as well skip the word `var` by simply using `emptyInput := "empty input"` respectively.

Reply:
As discussed, the vars need to be global. I did rename them to camel-case.